examples/EvaluationDataToApplicationInsights/Pages/Index.cshtml.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.ApplicationInsights;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.FeatureManagement;
+
+namespace EvaluationDataToApplicationInsights.Pages
+{
+    public class IndexModel : PageModel
+    {
+        private readonly IVariantFeatureManager _featureManager;
+        private readonly TelemetryClient _telemetry;
+
+        public string? Username { get; set; }
```

Comment:
```suggestion
        public string Username { get; set; }
```

Reply:
Good point, I removed usage of nullable.